rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -426,6 +426,51 @@ rocalCropResizeFixed(
     return output;
 }
 
+RocalTensor ROCAL_API_CALL
+rocalResizeCropFixed(
```

Comment:
Can we not use CropResizeFixed

Reply:
rocalCropResizeFixed is an already existing API

Reply:
Yes, but can't we just add the aditional params to the existing API instead of creating a new one

Reply:
Changed the API name to rocalROIResize